Sources/SBPlatformDestination.swift

```diff
@@ -468,11 +470,19 @@ public class SBPlatformDestination: BaseDestination {
         var details = [String: String]()
 
         details["os"] = OS
-        let osVersion = ProcessInfo.processInfo.operatingSystemVersion
-        // becomes for example 10.11.2 for El Capitan
-        var osVersionStr = String(osVersion.majorVersion)
-        osVersionStr += "." + String(osVersion.minorVersion)
-        osVersionStr += "." + String(osVersion.patchVersion)
+        
+        var osVersionStr: String
+        if #available(OSX 10.10, *) {
```

Comment:
why is this necessary? what would be if the OS is iOS?

Reply:
`ProccessInfo.operatingSystemVersion` is available only starting macOS 10.10. Asterisk after the comma means that this check will pass on all other platforms according to [the documentation](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html). Because minimum support for macOS is 10.9, this test will not pass only for this version and only on macOS, that's why it's safe to put "10.9" into `osVersionStr`. We won't know the patch version, but I believe that's not critical.

Reply:
I am unsure about that condition really works as intended and am concerned that we will break support for newer OSX versions and general iOS and Apple TV versions. I checked the Apple docs (thanks for the link!) but they did not make me smarter on that one.

Did you try out the condition and does it really work?